scripts/dotnet-cli-build/CompileTargets.cs

```diff
@@ -143,10 +143,10 @@ public static BuildTargetResult CompileCoreHost(BuildTargetContext c)
         [Target(nameof(CompileCoreHost))]
         public static BuildTargetResult PackageCoreHost(BuildTargetContext c)
         {
-            if (!string.Equals(Environment.GetEnvironmentVariable("BUILD_COREHOST_PACKAGES"), "1"))
-            {
-                return c.Success();
-            }
+//            if (!string.Equals(Environment.GetEnvironmentVariable("BUILD_COREHOST_PACKAGES"), "1"))
```

Comment:
In x86, we hit MAX_PATH in VSO. Keep this uber packaging variable to guard against this. I am experimenting few layouts that would make VSO happy.